src/components/Modeler.vue

```diff
@@ -333,25 +333,44 @@ export default {
      * component is already registered, it is replaced by the new one.
      */
     registerInspectorExtension(node, config) {
-      const registeredIndex = node.inspectorConfig[0].items.findIndex(item => {
-        return config.id && config.id === item.id;
-      });
-        
-      let items;
-      if (config.container) {
-        // if it is a container, don't put it inside the default container
-        items = node.inspectorConfig[0].items;
+      this.addToInspector(
+        this.inspectorItemsToAddTo(node, config),
+        this.existingConfigIndex(node, config),
+        config
+      );
+    },
+    addToInspector(inspectorItems, existingIndex, config) {
+      if (existingIndex === -1) {
+        inspectorItems.push(config);
       } else {
-        // if it is not a container, put it in the default container
-        items = node.inspectorConfig[0].items[0].items;
+        inspectorItems[existingIndex] = config;
       }
-
-      if (registeredIndex === -1) {
-        items.push(config);
+    },
+    existingConfigIndex(node, config) {
+      return this.inspectorItems(node).findIndex(item => {
+        return config.id && config.id === item.id;
+      });
+    },
+    isContainer(config)
+    {
```

Comment:
To keep things consistent, place the opening bracket on the same line as the function name.